src/charm.py

```diff
@@ -351,7 +351,8 @@ def _prometheus_config(self) -> dict:
             The dict representing the config
         """
         return {
-            "prometheus": {
+            "metrics": {
```

Comment:
The [official example](https://grafana.com/docs/agent/latest/configuration/create-config-file/) has `remote_write` under `global` as well as under `configs`.
Any idea which one we need? Makes more sense to me that the `remote_write` section needs to be under `global`.

Reply:
Similarly, for the loki portion of the config, [looks like](https://grafana.com/docs/agent/latest/configuration/create-config-file/) we now need `logs` instead of `loki` for the section name.
Docs [seem partial](https://grafana.com/docs/agent/latest/configuration/create-config-file/#full-config-example) but would be nice to confirm the new format for `logs.scrape_configs` section supports what we currently have coded.

Reply:
`remote_write` under `global` sends _all_ metrics there. `metrics` is explicitly:

> The Prometheus subsystem config is useful for those migrating from Prometheus and _those who want to scrape metrics from something that currently does not have an associated integration._

^^ this is our use case (at least partly).

`loki` [is in fact deprecated](https://github.com/grafana/agent/blob/main/pkg/config/config.go#L102) the same as `metrics` (and `tempo`, which we don't use), but still works for now. There's probably a `DEPRECATION NOTICE:` in the container logs. We may as well change it, but it's ok for now

Reply:
I changed it.